lib/render.js

```diff
@@ -75,3 +74,20 @@ function scopeNameFromLang (highlighter, lang) {
 
   return name
 }
+
+// create a parser instance, add plugins, update rules, etc...
+function makeParser (mdOptions) {
+  var parser = MD(mdOptions).use(lazyHeaders).use(emoji, {shortcuts: {}})
+
+  // monkey patch the 'fence' parsing rule to restore markdown-it's pre-5.1 behavior
+  // (see https://github.com/markdown-it/markdown-it/issues/190)
+  var stockFenceRule = parser.renderer.rules.fence
+  parser.renderer.rules.fence = function (tokens, idx, options, env, slf) {
```

Comment:
@ashleygwilliams this pattern here is basically the same thing [markdown-it-expand-tabs](https://github.com/revin/markdown-it-expand-tabs/blob/master/index.js#L12-L16) does. If you think it would be cleaner, I could extract this into a super tiny markdown-it plugin. We could either publish it separately or just include it with marky in a separate module, and use markdown-it's regular `.use()` API to hook it up.